### DISCUSSION TITLE
New translation in Swahili

### DISCUSSION THREAD

**Discussion by Hans4M** [sircharlo/meeting-media-manager#2496] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak, Swahili (SW/sw).

**Comment by Hans4M** [comment]
I would like to help translate M³ into a language I speak, Swahili (SW/sw).

**Reply by sircharlo** [reply]
Hi @Hans4M  ! 
Sorry I took so long to answer. We're busy working on the new version on M³, so this version is on maintenance mode for now. 
Keep your eyes peeled for the newest version, which will take care of all these bugs along with making managing media much, much easier and intuitive. 
We'll eventually need help to translate that version as well. We'll keep you posted. 
Thanks for your patience!

**Reply by Hans4M** [reply]
Thanks for the update an keeping me in the loop. I understand that you have been busy and looking forward to the new updates. I'm ready to help when you have everything set.

**Comment by Hans4M** [comment]
Hi @sircharlo 
I see you've rolled out a new update of the app. Was asking if i can begin assisting with translation to Swahili. Your efforts are much appreciated. Thank you and the brothers and sisters in the team.

**Reply by sircharlo** [reply]
Thank you so much for your willing spirit! The translation website was updated with new strings to prepare for the updated app's release.
If you would like to volunteer to become an M³ language moderator for a language you speak, please [sign into Crowdin](https://crowdin.com/project/meeting-media-manager) using your GitHub account, and then let me know once that is done.

As language moderator, you will get notified of changes to the English original strings, have a nice and easy to use interface to review existing as well as untranslated strings in your language, and submit new translations to the repo easily.

**Reply by sircharlo** [reply]
Take note, the file that contains the strings to translate for the new app is in the folder called `new-strings`.

**Reply by Hans4M** [reply]
I've logged in successfully . looking forward to beginning. Thanks
